pkg/amqplib/AmqpConsumer.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace Enqueue\Amqplib;
+
+use Interop\Queue\InvalidMessageException;
+use Interop\Queue\PsrConsumer;
+use Interop\Queue\PsrMessage;
+use PhpAmqpLib\Channel\AMQPChannel;
+use PhpAmqpLib\Message\AMQPMessage as LibAMQPMessage;
+use PhpAmqpLib\Wire\AMQPTable;
+
+class AmqpConsumer implements PsrConsumer
+{
+    private $channel;
+    private $queue;
```

Comment:
we use php-cs-fixer to make sure the code looks similar. Could you please use it too? https://github.com/php-enqueue/enqueue-dev/blob/master/docs/contribution.md